automem/api/memory.py

```
@@ -681,4 +682,221 @@ def associate() -> Any:
+        embeddings: List[Optional[List[float]]] = [None] * len(validated)
+
+        if generate_real_embeddings_batch:
+            try:
+                embeddings = generate_real_embeddings_batch(contents)
+            except Exception:
+                logger.exception("Batch embedding failed, falling back to individual")
+                for j, c in enumerate(contents):
+                    try:
+                        embeddings[j] = generate_real_embedding(c)
+                    except Exception:
+                        pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)